packages/vue3-vite/src/util/defaults.ts

```diff
@@ -38,53 +38,56 @@ export const CypressDevDependencies = {
 };
 
 export const VitestDevDependencies = {
-  '@nrwl/vite': '^15.4.2',
+  '@nrwl/vite': '^15.4.4',
   '@vue/test-utils': '^2.2.6',
   'happy-dom': '^8.1.1',
-  vitest: '^0.25.8',
+  vitest: '^0.26.3',
 };
 
 export const JestDevDependencies = {
+  '@babel/preset-env': '^7.20.2',
+  '@jest/globals': jestGlobalsVersion,
   '@vue/vue3-jest': vue3JestVersion,
   'jest-environment-jsdom': '^29.3.1',
 };
 
+export const LintDevDependencies = {
```

Comment:
Moved lint dependencies here so it can be included in both app and lib. Technically a user might only generate a library, which currently breaks the executors since these deps are missing.